README.md

```diff
@@ -173,6 +173,10 @@ Inside your app, you use icons to represent an action, such as copying text or n
 Icon(FluentIcons.add),
 ```
 
+For a complete reference of current icons, please check the [online demo](https://bdlukaa.github.io/fluent_ui/) and click on "Icons".
```

Comment:
For future reference: I was thinking to use Navigator 2.0 to be able to do 

https://bdlukaa.github.io/fluent_ui/icons/

Reply:
This would be great in terms of SEO & visibility of the whole package, like a lead magnet (_look here for a cute list of Fluent icons_)

I do recommend you change to Navigator 2.0 to make this happen